### ISSUE TITLE
[Bug] Tutorial AttributeError at indexing dataset

### ISSUE THREAD

**Issue by baehr95** [delira-dev/delira#81]
**Description**
tutorial throws AttributeError

**Environment**
* OS: ubuntu 18.04
* Python version: 3.7.1
* `delira` version 0.3.1
* How did you install `delira`? [ pip | source | conda | docker ]
pip

**Reproduction**
```python
from delira.data_loading import TorchvisionClassificationDataset
dataset_train = TorchvisionClassificationDataset("mnist", train=True,img_shape=(224, 224))
first_img=dataset_train[0]
```
**Additional context**

```bash
AttributeError                            Traceback (most recent call last)
 in 
----> 1 dataset_train[0]

/work/scratch/baehr/anaconda3/envs/delira/lib/python3.7/site-packages/delira/data_loading/dataset.py in __getitem__(self, index)
    772             data = self.data[index]
    773             data_dict = {"data": np.array(data[0]),
--> 774                          "label": data[1].numpy().reshape(1).astype(np.float32)}
    775 
    776             if self.one_hot:

AttributeError: 'int' object has no attribute 'numpy'
```

**Comment by justusschock** [comment]
This is fixed as a part of #66 but not yet merged into master. The problem is an update of torchvision, making the internal torchvision dataset returning ints instead of scalar tensors

**Comment by justusschock** [comment]
Edit: for now, you can simply downgrade your torchvision to 0.2.1 via `pip install torchvision==0.2.1`
